.github/workflows/release-ts-api-package.yml

```diff
@@ -1,16 +1,133 @@
+        if: startsWith(runner.name, 'GitHub Actions')
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: true
+          swap-storage: false
+          large-packages: false
+
+      - uses: actions/checkout@v4
+        
+      - name: Pull docker image optinally
+        run: |
+          docker pull litentry/litentry-parachain:latest
+
+      - name: Save docker image
+        run: |
+          docker save litentry/litentry-parachain:latest | gzip > litentry-parachain-dev.tar.gz
+
+      - name: Upload docker image
+        uses: actions/upload-artifact@v3
+        with:
+          name: litentry-parachain-dev
+          path: litentry-parachain-dev.tar.gz
+          if-no-files-found: error
+
+      - name: Fail early
+        if: failure()
+        uses: andymckay/cancel-action@0.3
+
+  tee-build:
+    runs-on: ubuntu-latest
+    needs:
+      - parachain-build-dev
+    steps:
+      - name: Free up disk space
+        if: startsWith(runner.name, 'GitHub Actions')
+        uses: jlumbroso/free-disk-space@main
+        with:
+          tool-cache: true
+          swap-storage: false
+          large-packages: false
+
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+
+          driver: docker
+
+      - name: Pull and tag worker and cli image optionally
+        run: |
+          docker pull litentry/litentry-worker:latest
+          docker pull litentry/litentry-cli:latest
+
+      - run: docker images --all
+
+      - name: Save docker images
+        run: docker save litentry/litentry-worker:latest litentry/litentry-cli:latest | gzip > litentry-tee.tar.gz
+
+      - name: Upload docker images
+        uses: actions/upload-artifact@v3
+        with:
+          name: litentry-tee
+          path: litentry-tee.tar.gz
+          if-no-files-found: error
+
+      - name: Fail early
+        if: failure()
+        uses: andymckay/cancel-action@0.3 
+        
+  update-publish:
+    runs-on: ubuntu-latest
+    needs:
+      - parachain-build-dev
+      - tee-build
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Pull polkadot image
+        run: |
+          docker pull parity/polkadot
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: litentry-parachain-dev
+
+      - uses: actions/download-artifact@v3
+        with:
+          name: litentry-tee
+
+      - name: Load docker image
+        run: |
+          docker load < litentry-parachain-dev.tar.gz
+          docker load < litentry-tee.tar.gz
+          docker images
+
+      - name: Enable corepack and pnpm
+        run: corepack enable && corepack enable pnpm
+
+      - name: Generate parachain artefacts
+        run: |
+          ./tee-worker/scripts/litentry/generate_parachain_artefacts.sh
+          ls -l docker/generated-rococo/
+          ls -l tee-worker/docker/litentry/
+          shasum tee-worker/docker/litentry/*.json
```

Comment:
If I don't include it here, it will result in an error: 

`unable to prepare context: path "/home/runner/work/litentry-parachain/litentry-parachain/tee-worker/docker/litentry" not found.`
:p

Reply:
I mean these three lines:
```
          ls -l docker/generated-rococo/
          ls -l tee-worker/docker/litentry/
          shasum tee-worker/docker/litentry/*.json
```